src/osd/OSDMap.cc

```diff
@@ -1481,6 +1540,19 @@ int OSDMap::apply_incremental(const Incremental &inc)
       (*primary_temp)[p->first] = p->second;
   }
 
+  for (auto& p : inc.new_pg_remap) {
```

Comment:
FWIW this mirrors how we do {new,old}_blacklist and _pools.  But not erasure_code_profiles. 

We should probably be asserting in encode if the sets are not mutually exclusive and treat that as a bug in the monitor.